test/src/main/java/org/apache/accumulo/test/ComprehensiveTableOperationsIT.java

```diff
@@ -0,0 +1,957 @@
+    assertEquals(expected, ops.tableIdMap().keySet());
+  }
+
+  @Test
+  public void test_exists() throws Exception {
+    // Thoroughly tested elsewhere, but simple enough to test here. Test that all system tables
+    // and user tables exist
+    userTable = getUniqueNames(1)[0];
+    ops.create(userTable);
+    var expected = Arrays.stream(AccumuloTable.values()).map(AccumuloTable::tableName)
+        .collect(Collectors.toSet());
+    expected.add(userTable);
+    for (String table : expected) {
+      assertTrue(ops.exists(table));
+    }
+  }
+
+  @Test
+  public void test_create() throws Exception {
+    // Creating user tables is thoroughly tested. Make sure we can't create any of the already
+    // existing system tables, though.
+    for (var systemTable : AccumuloTable.values()) {
+      assertThrows(AccumuloException.class, () -> ops.create(systemTable.tableName()));
+    }
+  }
+
+  @Test
+  public void test_exportTable_importTable() throws Exception {
+    // exportTable, importTable for user tables is tested in ImportExportIT. Ensure test exists
+    assertDoesNotThrow(() -> Class.forName(ImportExportIT.class.getName()));
+    // exportTable, importTable untested for system tables. Test basic functionality here
+    userTable = getUniqueNames(1)[0];
+    ops.create(userTable);
+    Path baseDir = ImportExportIT.createBaseDir(getCluster(), getClass());
+    var fs = getCluster().getFileSystem();
+
+    // export the user table, so we can test importing into the system tables
+    ops.offline(userTable, true);
+    Path exportUserDir;
+    exportUserDir = new Path(baseDir, "export_userdir");
+    fs.deleteOnExit(exportUserDir);
+    ops.exportTable(userTable, exportUserDir.toString());
+
+    var sysTables = AccumuloTable.values();
+    for (int i = 0; i < sysTables.length; i++) {
+      var sysTable = sysTables[i];
+      Path exportDir = new Path(baseDir, "export" + i);
+      fs.deleteOnExit(exportDir);
+
+      // not offline, can't export
+      assertThrows(IllegalStateException.class,
+          () -> ops.exportTable(sysTable.tableName(), exportDir.toString()));
+
+      switch (sysTable) {
+        case ROOT:
+        case METADATA:
+          // can't offline, so will never be able to export
+          assertThrows(AccumuloException.class, () -> ops.offline(sysTable.tableName(), true));
+          assertThrows(AccumuloException.class,
+              () -> ops.importTable(sysTable.tableName(), exportUserDir.toString()));
+          break;
+        case FATE:
+        case SCAN_REF:
+          ops.offline(sysTable.tableName(), true);
+          try {
+            assertThrows(AccumuloException.class,
+                () -> ops.exportTable(sysTable.tableName(), exportDir.toString()));
+            assertThrows(AccumuloException.class,
+                () -> ops.importTable(sysTable.tableName(), exportUserDir.toString()));
+          } finally {
+            ops.online(sysTable.tableName(), true);
+          }
```

Comment:
This will be changed once #5433 is completed and merged in: should not be able to offline any system table

Reply:
#5433 has been merged in. This test has been updated (in 4eb99ebd482139c834dbad28948a1842ab88b7d5) to expect an exception when offlining any system table